src/components/trace/hat-trace.ts

```diff
@@ -432,12 +441,53 @@ export class HaAutomationTracer extends LitElement {
     return html`${entries}`;
   }
 
+  protected updated(props) {
+    super.updated(props);
+
+    // Pick first path when we load a new trace.
+    if (props.has("trace")) {
+      for (const el of Array.from(
+        this.shadowRoot!.querySelectorAll("ha-timeline")
+      )) {
+        if (el.dataset.path) {
+          fireEvent(this, "value-changed", { value: el.dataset.path });
+          this._selectedPath = el.dataset.path;
+          break;
+        }
+      }
+    }
+
+    this.shadowRoot!.querySelectorAll("ha-timeline").forEach((el) => {
```

Comment:
We are doing this for every update?

Reply:
Yeah but I guard if it's upgraded.

Reply:
We still query and run over it? Can't we just only do it on change of `trace`?

Reply:
actually, we can't. We still need to set the "active" color.

Reply:
that is `_selectedPath` right?